src/ProjectModule/Components/ProjectList/ProjectList.tsx

```diff
@@ -40,6 +43,7 @@ const closeModal = () => {
 
       console.log(response.data);
       setProjects(response.data.data);
+setIsLoading(false)     
```

Comment:
This should be in both the try and catch blocks, or only in the finally block. Currently, if there is an error, the application will stuck in the loading state.